src/main/java/pl/javastart/task/CoordinateSystem.java

```diff
@@ -6,6 +6,60 @@ public class CoordinateSystem {
 
     // uzupełnij metodę. Używaj scannera przekazanego w parametrze.
     void run(Scanner scanner) {
-        System.out.println("Punkt (5, -3)");
+        System.out.println("Podaj współrzędną X: ");
+        int x = scanner.nextInt();
+        System.out.println("Podaj współrzędną Y: ");
+        int y = scanner.nextInt();
+        Point point = new Point(x, y);
+        System.out.println("Punkt (" + x + ", " + y + ") " + getPointPosition(point));
+    }
+
+    String getPointPosition(Point point) {
+        String pointPosition = "";
+        if (isInFirstQuarter(point)) {
+            pointPosition = "leży w I ćwiartce układu współrzędnych";
+        } else if (isInSecondQuarter(point)) {
+            pointPosition = "leży w II ćwiartce układu współrzędnych";
+        } else if (isInThirdQuarter(point)) {
+            pointPosition = "leży w III ćwiartce układu współrzędnych";
+        } else if (isInFourthQuarter(point)) {
+            pointPosition = "leży w IV ćwiartce układu współrzędnych";
+        } else if (isInCentreOfCoordinateSystem(point)) {
+            pointPosition = "leży na środku układu współrzędnych";
+        } else if (isOnxAxis(point)) {
+            pointPosition = "leży na osi X";
+        } else if (isOnyAxis(point)) {
+            pointPosition = "leży na osi Y";
+        }
+
+        return pointPosition;
```

Comment:
Ten ostatni if-else to ostatnia możliwa pozycja, więc można go po prostu zamienić na else i wtedy nie będzie potrzebny na końcu zwracanie pustego napisu